### ISSUE TITLE
Fix: Down arrow Image instead of "X" arrow

### ISSUE THREAD

**Issue by Bhavnaharitsa** [openMF/mobile-wallet#769]
**Summary:**

![Screenshot_20200227-061701_Mifos Pay](https://user-images.githubusercontent.com/37215508/75401651-cc566780-5928-11ea-85fc-f1db1e258e5b.jpg)
 This particular image confuses me to click it. When I click it, nothing happens. Later, I learnt it is to swipe. So we can add a swipe down arrow image so that user will swipe down.

**Comment by haran2248** [comment]
Is this issue solved?Can i work on this?

**Comment by Bhavnaharitsa** [comment]
From my side, since I opened this issue it's a yes, but anyway confirm it with @devansh-299 before proceeding :)

**Comment by devansh-299** [comment]
@Bhavnaharitsa @haran2248 Instead of having a down arrow, I think it would be better to automatically collapse the bottom sheet, let say after 3 secs.
What do you think?

**Comment by Bhavnaharitsa** [comment]
@devansh-299 Yes, that sounds better tooo and also 3 seconds is ideal. But we need a timer for it.
@haran2248 If you are taking up that, please read some docs on that

**Comment by haran2248** [comment]
In that case why dont we just use a toast message for this?Why do we confuse the user with the bottom sheet?

**Comment by devansh-299** [comment]
> In that case why dont we just use a toast message for this?Why do we confuse the user with the bottom sheet?

The transaction making process is already being done using the bottom sheet

**Comment by therajanmaurya** [comment]
Fixed in compose implementation
